Assets/SEE/GraphProviders/GitGraphProvider.cs

```diff
@@ -0,0 +1,282 @@
+using Cysharp.Threading.Tasks;
+using LibGit2Sharp;
+using SEE.DataModel.DG;
+using SEE.Game.City;
+using SEE.Utils.Config;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+
+namespace SEE.GraphProviders
+{
+    /// <summary>
+    /// Calculates metrics between 2 revisions from a git repository and adds these to a graph.
+    /// </summary>
+    [Serializable]
+    public class GitGraphProvider : GraphProvider
+    {
+        /// <summary>
+        /// The path to the VCS containing the two revisions to be compared.
+        /// </summary>
+        private string VCSPath = string.Empty;
+
+        /// <summary>
+        /// The older revision that constitutes the baseline of the comparison.
+        /// </summary>
+        private string OldRevision = string.Empty;
+
+        /// <summary>
+        /// The newer revision against which the <see cref="OldRevision"/> is to be compared.
+        /// </summary>
+        private string NewRevision = string.Empty;
+
+        /// <summary>
+        /// Calculates metrics between 2 revisions from a git repository and adds these to <paramref name="graph"/>.
```

Comment:
Numbers until (including) twelve are generally given in words. So, "2" should be replaced by "two".

Reply:
Was fixed.